src/libs/IOUUtils.js

```diff
@@ -32,10 +32,10 @@ function calculateAmount(numberOfParticipants, total, isDefaultUser = false) {
  * @param {String} actorEmail
  * @param {Number} amount
  * @param {String} currency
- * @param {String} type
+ * @param {String} isDeleting - whether the user is deleting the request
  * @returns {Object}
  */
-function updateIOUOwnerAndTotal(iouReport, actorEmail, amount, currency, type = CONST.IOU.REPORT_ACTION_TYPE.CREATE) {
+function updateIOUOwnerAndTotal(iouReport, actorEmail, amount, currency, isDeleting = false) {
```

Comment:
Curious why isDeleting is preferred to just passing the type?

Reply:
no reason, i just thought it was clearer to pass a bool and avoid the comparison.